### PR TITLE
AB#78554 turn on nullable for ahp www

### PR DESCRIPTION
**Pull request by Carq**

### CODE REVIEW

File: src/HE.Investment.AHP.WWW.Tests/Views/Scheme/FundingTests.cs [hunk 1]
```diff
@@ -12,12 +13,13 @@ public class FundingTests : ViewTestBase
     private const string FundingError = "Funding error";
     private const string HousesError = "Houses error";
     private static readonly SchemeViewModel Model = TestSchemeViewModel.Test;
+    private readonly RouteData _routeData = new(new RouteValueDictionary { { "applicationId", "123" }, { "homeTypeId", "456" }, });
```
Comment (jjankowski87) [inline src/HE.Investment.AHP.WWW.Tests/Views/Scheme/FundingTests.cs:16]: you don't need homeTypeId for scheme

File: src/HE.Investment.AHP.WWW/Views/Application/ApplicationSections.cs [hunk 2]
```diff
@@ -43,10 +43,11 @@ public static IList<TaskListSectionModel> CreateSections(IList<ApplicationSectio
                     false),
             }
             .Where(i => i != null)
+            .Cast<TaskListSectionModel>()
```
Comment (jjankowski87) [inline src/HE.Investment.AHP.WWW/Views/Application/ApplicationSections.cs:46]: maybe .Select(x => x!) is better?

File: src/HE.Investments.Common.WWW/Components/WorkflowBackButton/WorkflowBackButton.cshtml [hunk 3]
```diff
@@ -14,8 +15,8 @@
 
         if (isSubSection)
         {
-            ViewContext.RouteData.Values.TryGetValue(Model.ParamName, out object paramValue);
-            routeData.Add("id", paramValue?.ToString());
+            ViewContext.RouteData.Values.TryGetValue(Model.ParamName, out object? paramValue);
```
Comment (jjankowski87) [inline src/HE.Investments.Common.WWW/Components/WorkflowBackButton/WorkflowBackButton.cshtml:18]: I've fixed this some time ago and it caused app crashes, have you tested it?

**Review by jjankowski87 — APPROVED** [review]

File: src/HE.Investments.Common.WWW/Components/WorkflowBackButton/WorkflowBackButton.cshtml (hunk 3 above)
Comment (Carq) [reply src/HE.Investments.Common.WWW/Components/WorkflowBackButton/WorkflowBackButton.cshtml:18]: Yes.

File: src/HE.Investment.AHP.WWW.Tests/Views/Scheme/FundingTests.cs (hunk 1 above)
Comment (Carq) [reply src/HE.Investment.AHP.WWW.Tests/Views/Scheme/FundingTests.cs:16]: Fixed.

File: src/HE.Investment.AHP.WWW/Views/Application/ApplicationSections.cs (hunk 2 above)
Comment (Carq) [reply src/HE.Investment.AHP.WWW/Views/Application/ApplicationSections.cs:46]: IMO - doesn't matter.

**Review by dawid-pawulski — APPROVED** [review]

**Pull request merged**